src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -25,6 +25,7 @@ def call(_options = {})
       end
     end
 
+    report_to_scm({})
```

Comment:
We don't have access to the filters here, but is it necessary?

When it is called it breaks with:

```
33a915bc-d8d5-4fe3-9f80-ae427817a5d3] [52:2015.68]   
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] NoMethodError (undefined method `start_with?' for nil:NilClass):
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3]   
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/concerns/multibuild_package.rb:11:in `striping_multibuild_suffix'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/package.rb:157:in `get_by_project_and_name'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:34:in `target_package'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:99:in `multibuild_flavors'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:75:in `target_package_names'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:171:in `block (2 levels) in report_to_scm'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:170:in `block in report_to_scm'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step.rb:168:in `report_to_scm'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow/step/configure_repositories.rb:29:in `call'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow.rb:34:in `block in call'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow.rb:33:in `each'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/workflow.rb:33:in `call'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/token/workflow.rb:20:in `each'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/token/workflow.rb:20:in `call'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/controllers/trigger_workflow_controller.rb:12:in `block in create'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/models/user.rb:864:in `run_as'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] app/controllers/trigger_workflow_controller.rb:11:in `create'
[33a915bc-d8d5-4fe3-9f80-ae427817a5d3] config/initializers/wrap_parameters.rb:37:in `call'
```